src/pages/ProjectsPageTemp.js

```diff
@@ -1,27 +1,18 @@
 import React from 'react';
+import '../styling/ProjectsPageTemp.css';
 
 const ProjectsPageTemp = () => (
   <div class="container mt-5">
-    <div class="row text-center">
-      <div class="col"></div>
-      <div class="col-8">
+    <div class="row d-flex justify-content-center text-center">
+      <div class="col-md-8">
         <h1 class="mt-5">Projects Coming Soon</h1>
-        <h4 class="text-white mb-5">Stay tuned for our first projects!</h4>
-      </div>
-      <div class="col"></div>
-    </div>
-
-    <hr class="solid" />
+        <p> We are in the process of reviewing projects and will be posting projects shortly.
+            Check back in a couple weeks to see newly added projects.
+        </p>
+         <a href="http://eepurl.com/g9JPtn" target="_blank" class="btn btn-lg button mt-5" aria-disabled="false"> Notify Me About Projects </a>
```

Comment:
we should use `primary-button` or `secondary-button` for this

Reply:
done!